hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/DefaultRecordStore.java

```diff
@@ -662,27 +662,27 @@ public Data readBackupData(Data key) {
     }
 
     @Override
-    public MapEntrySet getAll(Set<Data> keys) {
+    public MapEntries getAll(Set<Data> keys) {
         checkIfLoaded();
         final long now = getNow();
 
-        final MapEntrySet mapEntrySet = new MapEntrySet();
```

Comment:
actually, the scope of `loadedEntries` is extremely small. It's fine as it is now.